src/Components/Events/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {View , StyleSheet, Text} from 'react-native';
+import EventCard from './Cards';
+import {getevents_highlights} from './../../content/events_and_highlights';
+import SectionSubheader from './../SectionSubheader';
+
+const events_highlight = getevents_highlights();
+
+function Events () {
```

Comment:
Use arrow function here for consistency as it's being used everywhere else.

Reply:
there is no arrow function in other component index.js so i have also made the same .

Reply:
Ok, leave it like this only then. Sorry for this.